composer.json

```diff
@@ -13,7 +13,12 @@
         "ext-swoole": ">=4.5",
         "casbin/casbin": "^2.2",
         "easyswoole/orm": "^1.4",
-        "easyswoole/easyswoole": "3.x"
+        "easyswoole/easyswoole": "~3.1|~3.2"
```

Comment:
The frame version is 3.4.x.

Reply:
Oh, yes. It should be ~3.4|~3.3. I accidentally spelled it wrong.